src/pages/_app.tsx

```diff
@@ -24,7 +24,7 @@ const App = ({ Component, pageProps }: AppPropsWithLayout) => {
           }
         `}
       </style>
-      <ChakraProvider theme={theme}>
+      <ChakraProvider theme={theme} colorModeManager={localStorageManager}>
```

Comment:
`localStorageManager` should not need to be imported as it is used by default for `colorModeManager`.
[colorModeManager under the hood](https://github.com/chakra-ui/chakra-ui/blob/b6befea762b44f923af42792473963215dc50ed1/packages/components/color-mode/src/color-mode-provider.tsx#L40)

Is there something that is not working without declaring it here?

Reply:
We had this same issue in other NextJS sites with Chakra and solved the same way, seems for NextJS is required to set `colorModeManager` (https://chakra-ui.com/docs/styled-system/color-mode#add-colormodemanager-optional-for-ssr)